packages/@aws-cdk/aws-lambda-nodejs/lib/build.ts

```diff
@@ -72,7 +73,7 @@ export function build(options: BuildOptions): void {
         : [],
     ].filter(Boolean) as string[];
 
-    const parcel = spawnSync('parcel', args);
+    const parcel = spawnSync('npx', args);
```

Comment:
```suggestion
    const parcelPkgPath = require.resolve('parcel-bundler/package.json'); // eslint-disable-line @typescript-eslint/no-require-imports
    const parcelDir = path.dirname(parcelPkgPath);
    const parcelPkg = require(parcelPkgPath); // eslint-disable-line @typescript-eslint/no-require-imports
    const binPath = path.join(parcelDir, parcelPkg.bin.parcel);
    
    const parcel = spawnSync(binPath, args);
```

Reply:
@eladb can you have a look?

Reply:
I will try this out, thanks for the suggestion!

Reply:
that worked for me, and I think it's the right way to go, I'll update this PR

Reply:
Yes this is how I would do it

Reply:
I updated the PR with that suggestion.